### PR TITLE
reading namespace from parameter

### PR DESCRIPTION
**Pull request by pabloinigoblasco**
### Description

The ros_control_namespace cant be used with the current code.
It was needed to execute a dual arm demo using two separated namespaces for ros2_control controller manager and move_group node.

### CODE REVIEW

File: moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp [hunk 1]
```diff
@@ -258,6 +258,12 @@ class Ros2ControlManager : public moveit_controller_manager::MoveItControllerMan
       if (!node_->has_parameter("ros_control_namespace"))
         ns_ = node_->declare_parameter<std::string>("ros_control_namespace", "/");
```
Comment (AndyZe) [inline moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp:259]: I like your change   :+1:

It seems like this should be added too, for completeness:

```suggestion
      if (!node_->has_parameter("ros_control_namespace"))
        ns_ = node_->declare_parameter<std::string>("ros_control_namespace", "/");
      else
        node_->get_parameter<std::string>("ros_control_namespace", ns_);
```

**Review by AndyZe — APPROVED** [review]

**Review by henningkayser — COMMENTED** [review]
I'm somewhat confused about this parameter. Is there a scenario where `ns` from the constructor is actually being used? If we ignore the value anyway, can't we just do a single call of  `ns_ = node_->declare_parameter<std::string>("ros_control_namespace", "/");`?

**Comment by pabloinigoblasco** [comment]
just to add more context.

These changes are needed to run this example with multiple robots.
https://lnkd.in/dAVhc6RJ

Right now to make this run I need the following repository configuration:
https://github.com/robosoft-ai/SMACC2/blob/feature/humle_dual_arm_demo/SMACC2-not-released.humble.repos

See more about this example here:-
https://discourse.ros.org/t/smacc-demonstrates-concurrent-control-of-two-arms-in-ros-2-using-moveit2-and-ros2-control/28793

**Comment by sjahr** [comment]
@pabloinigoblasco Can you format the code with `pre-commit run -a` (more information can be found [here](https://github.com/ros-planning/moveit2_tutorials#formatting-and-style)) and fix clang-tidy?

```
  Filtering for files that actually changed since main
  run clang-tidy for 1/4 file(s) in 2 process(es).
  /home/runner/work/moveit2/moveit2/.work/target_ws/src/moveit2/moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp:258:58: warning: statement should be inside braces [readability-braces-around-statements]
        if (!node_->has_parameter("ros_control_namespace"))
                                                           ^
                                                            {
  /home/runner/work/moveit2/moveit2/.work/target_ws/src/moveit2/moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp:258:58: note: FIX-IT applied suggested code changes
  /home/runner/work/moveit2/moveit2/.work/target_ws/src/moveit2/moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp:260:7: note: FIX-IT applied suggested code changes
        else
        ^
  /home/runner/work/moveit2/moveit2/.work/target_ws/src/moveit2/moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp:260:11: warning: statement should be inside braces [readability-braces-around-statements]
        else
            ^
             {
  /home/runner/work/moveit2/moveit2/.work/target_ws/src/moveit2/moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp:260:11: note: FIX-IT applied suggested code changes
  /home/runner/work/moveit2/moveit2/.work/target_ws/src/moveit2/moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp:261:73: note: FIX-IT applied suggested code changes
          node_->get_parameter<std::string>("ros_control_namespace", ns_);
                                                                          ^
  clang-tidy applied 4 of 4 suggested fixes.
  Suppressed 4478 warnings (4478 in non-user code).
'clang_tidy_check_moveit_ros_control_interface' returned with code '0' after 0 min 28 sec
```

**Review by vatanaksoytezer — APPROVED** [review]

**Pull request merged**